package.json

```diff
@@ -51,7 +51,7 @@
     ]
   },
   "devDependencies": {
-    "@datapunt/asc-assets": "^0.19.2-alpha.66",
+    "@datapunt/asc-assets": "^0.19.2-beta.5",
     "@datapunt/asc-ui": "^0.19.2-beta.5",
```

Comment:
Why are these listed both as `dependencies` as `devDependencies`? I would assume that `dependencies` would be enough.

Reply:
because they are listed in peerDependencies, so they won't be installed

Reply:
Peculiar, but ok :man_shrugging: